pkg/security/apis/security/v1/types.go

```diff
@@ -79,9 +79,11 @@ type SecurityContextConstraints struct {
 	ReadOnlyRootFilesystem bool `json:"readOnlyRootFilesystem" protobuf:"varint,17,opt,name=readOnlyRootFilesystem"`
 
 	// The users who have permissions to use this security context constraints
-	Users []string `json:"users,omitempty" protobuf:"bytes,18,rep,name=users"`
+	// +optional
```

Comment:
surprised this didn't regenerate the proto files

Reply:
oh, it did. surprised that didn't regenerate the fileDescriptorGenerated blob

Reply:
Generated strips comments (last I checked)